api-docs/list-services.md

```diff
@@ -0,0 +1,26 @@
+# List CU Services
```

Comment:
We're exposing the list of services so that the frontend can render a dynamic list of services instead of having to hard code them.